src/extension/tests/TestExtConfigSettingsHandler.py

```diff
@@ -240,6 +240,41 @@ def test_read_file_failures(self):
         self.assertRaises(Exception, ext_config_settings_handler.read_file, seq_no)
         shutil.rmtree(test_dir)
 
+    def test_read_all_config_settings_from_file(self):
+        ext_config_settings_handler = ExtConfigSettingsHandler(self.logger, self.json_file_handler, os.path.join(os.path.pardir, "tests", "helpers"))
+        seq_no = ext_config_settings_handler.get_seq_no()
+        config_settings = ext_config_settings_handler.read_file(seq_no)
+
+        # verify operation is read successfully
```

Comment:
To be pedantic, a successful read validates the value against the expected value. Not none only validates that there is some value present.